src/test/configs/compose-git-without-features/.devcontainer/docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+services:
+  app:
+    build: 
+      context: https://github.com/ShauryaAg/productapi-go.git
```

Comment:
Can we try not to use a personal repo? 
Feel free to use any repo in the https://github.com/devcontainers org.

Reply:
Changed to devcontainers/devcontainers.github.io repo.